packages/pages/src/components/link/link.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { Link } from "./link.js";
+
+export default {
+  title: "components/Link",
+  component: Link,
+} as ComponentMeta<typeof Link>;
+
+const Template: ComponentStory<typeof Link> = (args) => <Link {...args} />;
+
+// HREF
+export const HREF = Template.bind({});
+
+HREF.args = {
+  href: "https://yext.com",
+  children: "Learn More",
+};
+
+// CTA
+export const CTA = Template.bind({});
+
+CTA.args = {
```

Comment:
For my own understanding, when specifying args like this, would we still be able to override these props via controls in the UI?